### DISCUSSION TITLE
Align several sentences with NFA (Nemo Forced Aligner Tool)

### DISCUSSION THREAD

**Discussion by Ca-ressemble-a-du-fake** [NVIDIA-NeMo/NeMo#5847] [Q&A] [answered]
Hi,

First of all thanks for providing NFA which seems to work great with French.

I only tested it with one sentence and I would like to test it with several ones but I can't figure out how to input all the sentences I want it to align. The documentation states :

> By default, NFA needs to be provided with a 'manifest' file where each line specifies the absolute "audio_filepath" and "text" of each utterance that you wish to produce alignments for, like the format below:
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance"}

So I tried with one file and it worked but then how to align several utterances ?

This is what I tried :

```
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_1"}
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_2"}
```
```
{
"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_1"
"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_1"
}
```
```
{
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_1"},
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_2"}
}
```
```
{
[
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_1"},
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_2"}
]
}
```
None of these trials worked. 

How can I write several utterances in the manifest file ?

Thanks in advance for your help

**Comment by titu1994** [comment, marked as answer]
Your first example is the one which is correct - 

{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_1"}
{"audio_filepath": "/absolute/path/to/audio.wav", "text": "the transcription of the utterance_2"}

No new line at the end of the manifest file

**Reply by Ca-ressemble-a-du-fake** [reply]
Indeed! I may have added a new line at the end and that messed up everything. So this is really important not to add new line at the end! Thanks for your reply (I deleted my previous comment so that its clearer for the readers).

**Reply by titu1994** [reply]
@erastorgueva-nv can you add a check so this isn't an issue when reading the manifest ?
